src/test/resources/interfacewithunsolvedsymbols/input/com/example/Baz.java

```diff
@@ -0,0 +1,13 @@
+package com.example;
+
+import org.testing.UnsolvedType;
+
+public interface Baz<T> {
+    void doSomething(T value);
```

Comment:
Can we modify this test (or add another test) so that we also check that Specimin doesn't crash when the return type of an interface method that we have to keep is unsolved?